pkgs/development/lisp-modules-obsolete/quicklisp-to-nix-overrides.nix

```diff
@@ -4,8 +4,7 @@ let
   skipBuildPhase = x: {
     overrides = y: ((x.overrides y) // { buildPhase = "true"; });
   };
-  multiOverride = l: x: if l == [] then {} else
-    ((builtins.head l) x) // (multiOverride (builtins.tail l) x);
+  multiOverride = l: x: pkgs.lib.optionalAttrs (l != []) ((builtins.head l) x) // (multiOverride (builtins.tail l) x);
```

Comment:
I think missing parentheses here?

Reply:
Fixed in #237631

Reply:
OK I guess, though I really suspect that that if-then-else version was more readable.  Also supported by getting several +1 reviews without noticing that the new code parses wrong.  (_copied this from chat, for reference_)

Reply:
Possibly relevant to the proposal for `<|` in https://github.com/NixOS/rfcs/pull/148.